src/main/kotlin/org/wycliffeassociates/otter/jvm/app/widgets/takecard/events/AnimateDragEvent.kt

```diff
@@ -0,0 +1,14 @@
+package org.wycliffeassociates.otter.jvm.app.widgets.takecard.events
+
+import javafx.event.Event
+import javafx.event.EventType
+import javafx.scene.input.MouseEvent
+
+class AnimateDragEvent(
+    type: EventType<AnimateDragEvent>,
```

Comment:
Do you need to accept type in the constructor?  Can you just use the companion value?